src/app/features/components/main/main.scss

```
@@ -0,0 +1,355 @@
+@use '../../../shared/style/abstract/placeholders' as *;
+@use '../../../shared/style/abstract/constant' as *;
+@use '../../../shared/style/abstract/mixins' as *;
+
+.container {
+  max-width: $content-width;
+  margin: 0 auto;
+  padding: 0 7rem;
+  line-height: 1;
+
+  @include media-laptop-tablet {
+    padding: 0 2rem;
+  }
+
+  @include media-tablet {
+    padding: 0rem 2rem;
+  }
+
+  @include media-tablet-mobile {
+    padding: 1.5rem 1rem;
+  }
+}
+
+h2 {
+  font-size: 3.6rem;
+  font-weight: 500;
+}
+
+.description {
+  background-color: var(--mat-sys-surface-container-low);
+
+  &__container {
+    padding: 10rem 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    column-gap: 3rem;
+
+    @include media-tablet-mobile {
+      flex-direction: column;
+      row-gap: 2rem;
+      text-align: center;
+      padding: 5rem 0;
+    }
+  }
+
+  &__information {
+    display: flex;
+    flex-direction: column;
+    row-gap: 4rem;
+    width: 530px;
+
+    @include media-tablet-mobile {
+      width: 350px;
+    }
+
+    &__title {
+      font-size: 6.4rem;
+      font-weight: 800;
+
+      @include media-tablet {
+        font-size: 3.6rem;
+      }
+    }
+
+    &__text {
+      font-size: 1.8rem;
+      line-height: 1.5;
+
+      @include media-tablet {
+        font-size: 1.6rem;
+      }
+    }
+
+    &__button {
+      border-radius: 50px;
+      padding: 1rem;
+      font-size: 1.6rem;
+      width: 300px;
+      box-sizing: border-box;
+      @extend %transition;
+
+      &:hover {
+        cursor: pointer;
+      }
+
+      @include media-tablet {
+        align-self: center;
+      }
+    }
+  }
+
+  &__image {
+    background-image: url('../../../../assets/images/foxflow_description.png');
+    width: 530px;
+    height: 320px;
+    object-fit: cover;
+    background-repeat: round;
+
+    @include media-tablet {
+      width: 330px;
+      height: 220px;
+    }
+  }
+}
+
+.goals {
+  padding: 8rem 0;
+
+  &__title {
+    text-align: center;
+  }
+
+  &__container {
+    display: flex;
+    justify-content: space-around;
+    padding: 6rem 0 0;
+    align-items: center;
+
+    @include media-tablet {
+      flex-direction: column;
+      row-gap: 1rem;
+    }
+  }
+
+  &__image {
+    background-image: url('../../../../assets/images/goals.png');
+    background-repeat: round;
+    width: 40rem;
+    height: 40rem;
+    object-fit: cover;
+
```

When resizing the browser window or testing responsiveness with DevTools, the image tends to shrink.
Could you please try to keep the proportions when resizing the image?

```suggestion
  &__image {
    background-image: url('../../../../assets/images/goals.png');
    width: 40rem;
    height: 40rem;
    object-fit: cover;
    
    background-size: cover;
    background-position: center;
    background-repeat: no-repeat;

```

---

done